### PR TITLE
Platformsh updates

### PR DESCRIPTION
**Pull request by Crell**
| Q               | A
| --------------- | ---
| Bug fix?        | yes |
| New feature?    | no |
| BC breaks?      | no |
| License         | MIT |

Thanks for including Platform.sh setup instructions!  I intend to link to them from our (Platform.sh) docs shortly.  I have a few improvements to suggest, though.

1) All of our sample code uses an application name of `app`, which makes it documentation more consistent.
2) The basic config page was using the old-style syntax for .platform.app.yaml.  We don't use that anymore.  Rather than try to update it in place I simply merged in the "Advanced" configuration (which uses the current syntax), as I don't see a reason to not give people a tuned-cache out of the box.
3) The application secret is important for security, so I folded that back into the basic page.  I don't think it's an "advanced" feature; rather, it's something everyone ought to be doing.

It's debatable if the cron section should fold into the basic configuration as well.  I could go either way there, but the above are definitely improvements that should be included.

### CODE REVIEW

File: docs/cookbook/platform-sh.rst [hunk 1]
```diff
@@ -51,62 +51,44 @@ This is how this file should look like for Sylius (tuned version of the default
             yarn: "*"
             gulp-cli: "*"
 
+
```
Comment (lchrusciel) [inline docs/cookbook/platform-sh.rst]: Redundant blank line

File: docs/cookbook/platform-sh.rst [hunk 2]
```diff
@@ -212,6 +200,8 @@ Remember to have it imported in the config:
     It is important to place newly created file after importing regular parameters.yml file. Otherwise your database connection will not work.
     Also this will be the file where you should set your required parameters. Its value will be fetched from environmental variables.
 
+The application secret is used in several places in Sylius and Symfony. Platform.sh allows you to deploy an environment for each branch you have, and therefore it makes sense to have a secret automatically generated by the Platform.sh system.  The last 3 lines in the sample above will use the Platform.sh-provided random value as the application secret.
```
Comment (lchrusciel) [inline docs/cookbook/platform-sh.rst]: `system.  The` -> `system. The`(one space instead of two)

**Comment by CoderMaggie** [comment]
Welcome to Sylius @Crell ;)
Let me ask @stefandoorn for review here :)

**Comment by stefandoorn** [comment]
_I'm not sure why the diff is way bigger than just the proposed changes._

Anyway, good to see a core Platform.sh member is aboard (@Crell) :-)

@CoderMaggie Looks good to me!

**Comment by CoderMaggie** [comment]
@Crell will you have a while to rebase your branch with `1.0` instead of master?
Your changes are fixing the 1.0 version, while `master` is the 1.1.0 candidate branch, that is why I have changed the base branch of your PR to `1.0` and we need a rebase.

**Comment by Crell** [comment]
2 whitespace fixes, one rebase.

**Comment by CoderMaggie** [comment]
@Crell it seems the rebase went not so good:

image

😢 , can you try again?

**Comment by pjedrzejewski** [comment]
Hi Crell, thank you for your help, @lchrusciel rebased your commits and the changes are merged in #8756. 👍

**Comment by Crell** [comment]
Yay!  Thanks folks.

**Comment by pjedrzejewski** [comment]
@Crell Thanks for your help! :) Hopefully more and more Sylius shops are hosted on Platform.sh.

**Comment by Crell** [comment]
We should discuss setting up a template repo for Sylius, as we have for many other projects.  Off topic here, but please ping me in the Platform.sh slack channel and let's discuss it.  It looks like you've already done most of the work with these docs anyway.

**Pull request closed without merging**